engineering/python-style-guide.md

```diff
@@ -0,0 +1,13 @@
+source{d} Python Style Guide
+
+1. Contributing to existing projects - do not change the style.
+2. [PEP8](https://www.python.org/dev/peps/pep-0008/) with 99 char line length limit
+3. Indentation: 4 whitespaces
+4. Naming: classes: **CamelCase** functions: **lower_underscore_case** arguments: **lower_underscore_case**
```

Comment:
Note that PEP8 uses the word `CapWords`, and `CamelCase` also appears as a synonym there.
It also mentions `lower_case_with_underscores` and it doesn't mention `snake_case`, so if we're going to leave this point, I'd stick to PEP8 names.